langgraph/prebuilt/tool_node.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+import json
+from typing import Any, Sequence, Union
+
+from langchain_core.messages import AIMessage, AnyMessage, ToolCall, ToolMessage
+from langchain_core.runnables import RunnableConfig
+from langchain_core.runnables.config import get_executor_for_config
+from langchain_core.tools import BaseTool
+
+from langgraph.utils import RunnableCallable
+
+
+def str_output(output: Any) -> str:
+    if isinstance(output, str):
+        return output
+    else:
+        try:
+            return json.dumps(output)
+        except Exception:
+            return str(output)
```

Comment:
maybe debug log?

Reply:
Not sure, lots of things in python/langchain (eg our good friend pydantic) aren't json serializable ?